taqueria-plugin-tezos-client/README.md

```diff
@@ -7,8 +7,8 @@ The Tezos Client plugin exposes the typecheck and simulate tasks. The typecheck
 ## Tasks
 
 The Tezos Client plugin provides the following tasks to Taqueria:
-- `typecheck [sourceFiles] [--sandboxName sandbox]`
-- `simulate <sourceFiles> <input> [--storage value] [--sandboxName sandbox]`
+- `taq typecheck <sourceFiles> [--sandboxName local]`
```

Comment:
For typecheck, sourceFile is actually optional, that's why I chose square brackets instead

Reply:
Got it!

Also, `taq typecheck` can take a list of filenames right? ([sourceFile] vs [sourceFiles])

Reply:
Yep it can